README.md

```diff
@@ -2,477 +2,19 @@ _The document you are reading now is targeted at developers wanting to use or co
-
-#### Storage repositories
-
-Two storage repositories are currently supported: Git and MongoDB. Each one can be used independently for versions and snapshots.
-
-##### Git
-
-```json
-{
-  …
-  "storage": {
-    "git": {
-      "path": "Versions database directory path, relative to the root of this project",
-      "publish": "Boolean. Set to true to push changes to the origin of the cloned repository at the end of every run. Recommended for production only.",
-      "snapshotIdentiferTemplate": "Text. Template used to explicit where to find the referenced snapshot id. Must contain a %SNAPSHOT_ID that will be replaced by the snapshot ID. Only useful for versions",
-      "author": {
-        "name": "Name to which changes in tracked documents will be credited",
-        "email": "Email to which changes in tracked documents will be credited"
-      }
-    }
-  }
-  …
-}
-```
-##### MongoDB
-
-```json
-{
-  …
-  "storage": {
-    "mongo": {
-      "connectionURI": "URI for defining connection to the MongoDB instance. See https://docs.mongodb.com/manual/reference/connection-string/",
-      "database": "Database name",
-      "collection": "Collection name"
-    }
-  }
-  …
-}
-```
-
-### Environment variables
-
-Environment variables can be passed in the command-line or provided in a `.env` file at the root of the repository. See `.env.example` for an example of such a file.
-
-- `SMTP_PASSWORD`: a password for email server authentication, in order to send email notifications.
-- `SENDINBLUE_API_KEY`: a SendInBlue API key, in order to send email notifications with that service.
-- `GITHUB_TOKEN`: a token with repository privileges to access the [GitHub API](https://github.com/settings/tokens).
-
-If an outgoing HTTP/HTTPS proxy to access the Internet is required, it is possible to provide it through the `HTTP_PROXY` and `HTTPS_PROXY` environment variable.
-
-## Deploying
-
-Deployment recipes are available in a [dedicated repository](https://github.com/OpenTermsArchive/deployment). Look at the [README](https://github.com/OpenTermsArchive/deployment#readme) to know how to deploy the engine.
-## Contributing
-
-### Getting a copy
-
-In order to edit the code of the engine itself, an editable and executable copy is necessary.
-
-First of all, follow the [requirements](#requirements) above. Then, clone the repository:
-
-```sh
-git clone https://github.com/OpenTermsArchive/engine.git
-cd OpenTermsArchive
-```
-
-Install dependencies:
-
-```sh
-npm install
-```
-
-### Testing
-
-If changes are made to the engine, check that all parts covered by tests still work properly:
-
-```sh
-npm test
-```
-
-If existing features are changed or new ones are added, relevant tests must be added too.
-
-### Suggesting changes
-
-To contribute to the core engine of Open Terms Archive, see the [CONTRIBUTING](CONTRIBUTING.md) file of this repository. You will need knowledge of JavaScript and Node.js.
+- - -
 
-### Sponsorship and partnerships
+## Table of contents
```

Comment:
I would also remove the table of contents because of the risk of forgetting to update it. I would replace it with a link to the global documentation.

Reply:
You are right, it will be difficult to manage, I deleted the table of contents.